server/src/test/java/io/crate/execution/engine/aggregation/impl/StdDevAggregationTest.java

```diff
@@ -56,72 +58,92 @@ private Object executeAggregation(DataType<?> argumentType, Object[][] data) thr
 
     @Test
     public void test_functions_return_type_is_always_double_for_any_argument_type() {
-        for (DataType<?> type : Stream.concat(
-            DataTypes.NUMERIC_PRIMITIVE_TYPES.stream(),
-            Stream.of(DataTypes.TIMESTAMPZ)).toList()) {
-
-            FunctionImplementation stddev = nodeCtx.functions().get(
-                null,
-                StandardDeviationAggregation.NAME,
-                List.of(Literal.of(type, null)),
-                SearchPath.pathWithPGCatalogAndDoc()
-            );
-            assertThat(stddev.boundSignature().returnType()).isEqualTo(DataTypes.DOUBLE);
+        for (var name: NAMES) {
```

Comment:
imho, this iteration here to use also the alias, is enough, I wouldn't do it on every test case.